tags/pages/actor-edit.tag.html

```diff
@@ -56,8 +88,8 @@ <h3>Domaines</h3>
       const loc = address.geometry
       loc.coordinates.reverse()
 
-      const actor = await api(`/actors`, {
-        method: 'POST',
+      const actor = await api(this.actorId ? `/actors/${this.actorId}` : '/actors', {
```

Comment:
Comme la route actuelle est `/actor/:id/edit`, j'hésiterais à faire l'édition `PUT` sur la même URL. 🎲
Celà dit, j'ai bien en tête les patterns d'URL RESTful.

C'est plus pour avoir partagé ma réflexion que pour proposer un changement.

Reply:
Merci pour le partage ! Plusieurs choses : 
 - Le PUT sur la même URL que la page d'édition en GET, c'est pas forcément recommandé. Mais de toute façon là, le PUT est fait sur une autre appli
 - Si on doit harmoniser IMO ce serait plutôt le front qui devrait changer au niveau des routes (l'API est "un peu plus" REST, ça nous fait un pattern de référence).
 - En gros, GET `/actors/123-abc/edit` (pour une page d'édition) et PUT `/actors/123-abc`, c'est ce que j'ai souvent vu en REST-like avec front HTML.

Un point intéressant là dessus (dont on avait déjà parlé) c'est les pluriels. Là c'est pas très dur de faire une harmonisation, il suffit de choisir une règle. Dans "Build API You Won't Hate" (https://apisyouwonthate.com/books/build-apis-you-wont-hate.html), qui est une référence comme une autre, ils utilisent le pluriel, et j'avoue que je préfère (beaucoup). Bref, postpone pour ça, mais point intéressant, ok?

Reply:
Effectivement, l'URL est sur l'API, donc la question n'existe pas 👍